src/tools/crossgen2/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -4,12 +4,11 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <AssemblyName>ILCompiler.ReadyToRun</AssemblyName>
-    <TargetFramework>netstandard1.3</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <DefineConstants>READYTORUN;$(DefineConstants)</DefineConstants>
     <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
     <Platforms>x64;x86</Platforms>
-    <OutputPath>$(BinDir)\crossgen2</OutputPath>
```

Comment:
Out of curiosity - what problems were you hitting with this binplacing? Seems like the Linux failure was unrelated to that. I kind of liked that we didn't have multiple copies of the DLLs and I didn't have to always wonder "am I running a stale copy?".

I can live with this change - just want to know what it fixes.

Reply:
I was hitting a segfault on process launch, that this fixed. But it seems that multiple other people weren't hitting this when they made changes, so possibly there was some sort of bad state on my machine.

Reply:
Ah, I figured it out. It wasn't actually crossgen2.dll being built overlapping itself, the issue is the other dlls being rewritten after crossgen2 publish when tibcmgr publish would occur. This would only occur in the branch with my custom changes. I'll try a slight variation on all of this that should address this issue. (I'll put the ILCompiler libraries into $(BinDir)/crossgen2/libs and only crossgen2 itself into $(BinDir)/crossgen. I think that will keep my scenario working as well as the convenient debug experience.